code/game/jobs/job/civilian.dm

```diff
@@ -91,7 +91,7 @@
 	selection_color = "#dddddd"
 	also_known_languages = list(LANGUAGE_CYRILLIC = 10, LANGUAGE_JIVE = 40, LANGUAGE_MONKEY = 20)
 	access = list(access_bar, access_kitchen, access_maint_tunnels, access_artist, access_theatre)
-
+	initial_balance = 600
```

Comment:
Go for small RNG instead of a set value (like default initial_balance)

Reply:
Isn't that the case? Club worker and manager define this the same

Reply:
Yeah it's the case, starting money is this initial_balance multiplied by a random coeff